docs/manual/job-options.md

```diff
@@ -446,31 +446,33 @@ The path must indicate a stored `password` entry in the storage facility.
 
 ![Storage Path for Secure Option](~@assets/img/jobs-options-secure-storage-path.png)
 
-## Remote option values
+## Option Model Provider
```

Comment:
Can we find a more descriptive way to describe this?  Happy to help brainstorm.

Reply:
yes - i'd like to change it too, but this is now the hyperlink that is used in the product to link to the docs. would need separate PR to change this and that

Reply:
We have ways to do redirects within the app.  I'll have to see if it will work with an anchor though.